### ISSUE TITLE
what is use of volume ftp/ftp is in dockerfile?

### ISSUE THREAD

**Issue by moinologics** [delfer/docker-alpine-ftp-server#59]
i see 
`VOLUME /ftp/ftp`
in dockerfile, is it required??

**Comment by delfer** [comment]
@moinologics hello! https://docs.docker.com/engine/reference/builder/#volume
No, it's not required. Do you have any issues with this?

**Comment by moinologics** [comment]
Yes, it's mount this volume even if we don't want to, so it will be good if we can remove

**Comment by delfer** [comment]
@moinologics can you please provide a PR? I would glad to merge it!

**Comment by QuiGonLeong** [comment]
@delfer PR created to remove the VOLUME /ftp/ftp 
#74 # # #
